### ISSUE TITLE
Attempts to add an image occlusion card result in "the first field is empty"

### ISSUE THREAD

**Issue by izaakm** [glutanimate/image-occlusion-enhanced#33]
1. I click the little image occlusion button and add an image
2. I add some occlusions/clozes to the image
3. I click the button (something like) "hide one, reveal all"
4. I click the "close" button in the bottom of the image editing window
5. Sometimes the image that I just added/edited isn't even shown on the "card"
6. I tried adding text to every field
7. Clicking the "add" at the bottom just gives me the error "the first field is empty"
8. Closing the window and going to "browse", I see that the "IO" cards have been added.
9. I can click open them, but I'm not able to edit the occlusions/clozes on the card ... the IO Enhanced button in the editor only gives me the option to "add" an image, and always opens a window to add a new card, so I'm not able to edit any of the occlusions that I've made as I see in the videos online.

**Comment by glutanimate** [comment]
Normally you'd stop after step 3. That's all you need to do to create an image occlusion card. Filing out the fields in Anki's regular "Add Cards" window won't work because this is a special note type.

To confirm that your cards were actually added after step 3 you can head to the card browser and click on "Added today" in the left sidebar. If your cards don't appear there something might have gone actually gone wrong and we can look into this further.

**Comment by glutanimate** [comment]
Sorry, I see now that you've already said that you can see the cards in the browser. In that case: I'm not exactly sure where the issue is. Step 9 should work properly if you click on the button from the card browser or when editing a card while reviewing. Could you go back into the card browser and try to click on the IO button again to see if you can edit an image occlusion card?

**Comment by izaakm** [comment]
1. I click the little image occlusion button and add an image
2. I add some occlusions/clozes to the image
3. I click the button, eg, "hide one, reveal all"
4. I click the "close" button in the bottom of the image editing window
5. **The image that I added/edited isn't shown in the  "Add" card window**

    This seems to be the expected behavior

6. I tried adding text to every field
7. Clicking "add" at the bottom just gives me the error "the first field is empty"

    At this point, the IO cards have been added even though nothing shows up on "add card" window. I should click "Close" to close the "add" window (normally I would need to click on "Add" to add the cards), and go to the browser to edit the IO cards.

8. Closing the window and opening the "Browser", I see that the "IO" cards have been added.

    When one of the IO cards is selected in the Browser, I see that the "Add Image Occlusion" button has changed to "Edit Image Occlusion", and I am able to edit it. 

So, the problem is just that using Anki's normal "Add" card window doesn't work for IO cards. But, given that this seems to be the expected behavior, I'll go ahead and close this issue.
